cmd/lukso/download.go

```diff
@@ -130,37 +151,97 @@ func (dependency *ClientDependency) createDir() error {
 	return err
 }
 
-func downloadBinaries(ctx *cli.Context) (err error) {
-	if !ctx.Bool(acceptTermsOfUseFlag) {
-		accepted := acceptTermsInteractive()
-		if !accepted {
-			return errors.New("You need to accept Terms to continue.")
-		}
+func installBinaries(ctx *cli.Context) (err error) {
+	if !cfg.Exists() {
+		log.Error("Folder not initialized - please make sure that you are working in initialized directory")
+
+		return
+	}
+
+	isRoot, err := isRoot()
+	if err != nil {
+		return err
 	}
-	if ctx.Bool(acceptTermsOfUseFlag) {
-		log.Info("You accepted Terms of Use provided by clients you want to download. You can read more here: https://github.com/prysmaticlabs/prysm/blob/develop/TERMS_OF_SERVICE.md")
+	if !isRoot {
+		return errNeedRoot
 	}
-	// Get os, then download all binaries into datadir matching desired system
-	// After successful download run binary with desired arguments spin and connect them
 
-	err = downloadGeth(ctx)
+	var (
+		consensusInput    string
+		executionInput    string
+		selectedConsensus string
+		selectedExecution string
+	)
+
+	consensusMessage := "Which consensus client you want to install?\n" +
+		"1: prysm\n> "
+	executionMessage := "Which execution client you want to install?\n" +
+		"1: geth\n> "
+
+	consensusInput = registerInputWithMessage(consensusMessage)
+	for consensusInput != "1" {
+		consensusInput = registerInputWithMessage("Please provide a valid option\n> ")
+	}
 
-	if nil != err {
-		return
+	switch consensusInput {
+	case "1":
+		selectedConsensus = prysmDependencyName
 	}
 
-	err = downloadValidator(ctx)
+	if selectedConsensus == lighthouseDependencyName {
+		log.Error("Please select different consensus client")
 
-	if nil != err {
-		return
+		return nil
 	}
 
-	err = downloadPrysm(ctx)
+	executionInput = registerInputWithMessage(executionMessage)
+	for executionInput != "1" {
+		executionInput = registerInputWithMessage("Please provide a valid option\n> ")
+	}
 
-	if nil != err {
-		return
+	switch executionInput {
+	case "1":
+		selectedExecution = gethDependencyName
 	}
 
+	if selectedExecution == erigonDependencyName {
+		log.Error("Please select different execution client")
+
+		return nil
+	}
```

Comment:
i think you can remove this from this PR. Let's not keep already "legacy" code here.